src/common_upgrades/change_macro_in_globals.py

```diff
@@ -0,0 +1,249 @@
+import os
+from xml.parsers.expat import ExpatError
+import re
+
+CONFIG_FOLDER = os.path.join("configurations", "configurations")
+COMPONENT_FOLDER = os.path.join("configurations", "components")
+IOC_FILE = "iocs.xml"
+GLOBALS_FILENAME = os.path.join("configurations", "globals.txt")
+
+# Matches an ioc name and it's numbered IOC's e.g. GALIL matches GALIL_01, GALIL_02
+FILTER_REGEX = "^{}(_[\d]{{2}})?$"
```

Comment:
A lot of these consts are used in a few places, can you put them into a shared file